libexec/rtld-elf/aarch64/rtld_c18n_asm.S

```diff
@@ -220,6 +220,30 @@ ENTRY(_rtld_tramp_hook)
 #endif
 END(_rtld_tramp_hook)
 
+ENTRY(_rtld_unw_getcontext)
+	/*
+	 * Get the address of the executive stack frame from the trusted frame
+	 * and derive an executive stack capability that the unwinder gets.
+	 */
+	ldr	c2, sealer_unwbuf
+	ldr	x10, [csp]
+	scvalue	c1, csp, x10
+	cseal	c1, c1, c2
```

Comment:
It seems that this is the only place where `sealer_unwbuf` is used in RTLD. Would it be a good idea to move `sealer_unwbuf` to `libunwind` and let it perform this `cseal`?

We can use linkage policy to ensure that only `libunwind` can call `_rtld_unw_getcontext`.